client/src/App.css

```diff
@@ -45,13 +45,27 @@
   grid-row-end: 1;
 }
 
-.project-collection {
+#project-collection-col-one {
```

Comment:
So, this seems like it's going to be hard to expand into four or five columns in the future. Could we perhaps have a single class for all the columns, and let them stack to the side as they're created?

Reply:
Yes, I'm going to say eventually. Right now we only have 3 columns, and this will require a lot of shifting. I'd like to change a lot of current css in the way the columns are set up